### ISSUE TITLE
[Feature] API call to get aggregated mailing statistics (count)

### ISSUE THREAD

**Issue by slackero** [xqueue/maileon-php-api-client#4]
As far I see there is no API call to request the aggregated statistics for a given mailingId as it's available under the page you get under a mailing report URL.
https://dev.maileon.com/api/rest-api-1-0/mailings/get-mailing-report-url/

It's just very inefficient to loop over every single possible count data (opener, clicks and so on) for every mailingId.

**Comment by mbeckerle-xqueue** [comment]
This is true. We have already quite some ideas to encance the reporting API and I will come back to this issue, once planning or even implementation is in a more concrete state I can talk about..

**Comment by ddietz-xqueue** [comment]
Hello @slackero ,

It's been a long time, but now a new resource for aggregated statistics is available:

https://support.maileon.com/support/get-mailing-summaries/

Best regards
